rollup.config.js

```diff
@@ -56,6 +56,7 @@ export default [
     input: './lib/model-viewer.js',
     output: {
       file: './dist/model-viewer.js',
+      sourcemap: true,
```

Comment:
Needed for debugging memory leaks, can remove and open another issue (valuable but we're building a lot of things per `npm run build` currently)

Reply:
:+1: !